js/src/data/reducer.js

```diff
@@ -233,14 +236,15 @@ const reducer = ( state = DEFAULT_STATE, action ) => {
 
 		case TYPES.RECEIVE_MC_ISSUES: {
 			const { query, data } = action;
-			const issues = state.mc_issues?.issues.slice() || [];
+			const issues =
+				state.mc_issues[ query.issue_type ]?.issues.slice() || [];
```

Comment:
We are sending now issue_type (account or product) as a parameter in order to update the issues